src/deployer.sol

```diff
@@ -171,17 +173,25 @@ contract Deployer {
         pile.rely(reception_);
         desk.rely(reception_);
     }
-    function deployLender(address currency_, address lenderfab_) public {
+
+    function deployLender(address currency_, address lenderfab_) public returns(address) {
         // LenderFab deploys a lender with the defined collateral and currency
         address lender_ = LenderFabLike(lenderfab_).deploy(currency_, address(collateral), address(lightswitch));
-        
+
         lender = LenderLike(lender_);
         lender.rely(god);
         lender.rely(address(desk));
-        
+
         desk.approve(lender_, uint(-1));
         pile.file("lender", lender_);
         desk.file("lender", lender_);
+        return lender_;
+    }
+
+    function deployLender(address currency_, address lenderfab_, address backer_) public returns(address) {
```

Comment:
Why not use inheritance here and create two deployers?

Reply:
Instead of two deployers I would initialize all variables of different LenderFabs in the lenderFab constructor and only have one `deploy` method without parameters.

Reply:
I think I still need inheritance. I will fix it in the next PR

Reply:
fixed. no inheritance needed. removed second deployLender method